### DISCUSSION TITLE
On-Premise ADS support

### DISCUSSION THREAD

**Discussion by JeroenKnoops** [danhellem/github-actions-issue-to-work-item#89] [Ideas]
We're using both On-Premise and Saas ADS. Does it make sense to add support for On-Premise ADS?
Do you accept a PR with these changes?

I see the `azure-devops-node-api` package can support both On-Premise and Saas ADS.

**Comment by danhellem** [comment]
Sorry @JeroenKnoops I did not notice you posted this.

I don't have any plans to try and support ADS.  This built mainly for the service. Feel free to fork and have fun :)
